inst/include/sampler.h

```diff
@@ -71,7 +71,7 @@ namespace smc {
         ///Structure used internally for resampling.
         arma::vec dRSWeights;
         ///Structure used internally for resampling.
-        arma::Col<unsigned int> uRSCount;
+        arma::Col<int> uRSCount;
```

Comment:
Thanks, that's good to know. Unfortunately I don't think this is the only variable name I've done this with.